lib/ueberauth/strategy/github/oauth.ex

```diff
@@ -28,36 +28,40 @@ defmodule Ueberauth.Strategy.Github.OAuth do
   These options are only useful for usage outside the normal callback phase of Ueberauth.
   """
   def client(opts \\ []) do
-    config = Application.get_env(:ueberauth, Ueberauth.Strategy.Github.OAuth)
+    client_module = Application.get_env(:ueberauth_github, :oauth2_client, OAuth2.Client)
```

Comment:
I'm sure theres a workaround that is more elegant/DRY but I havent gotten to toying with that yet